nima/webclient/webclient/src/main/java/io/helidon/nima/webclient/ClientConfig.java

```diff
@@ -66,4 +66,20 @@ public interface ClientConfig {
      */
     DnsAddressLookup dnsAddressLookup();
 
+    /**
+     * Whether to follow redirects.
+     *
+     * @return follow redirects
+     */
+    @ConfiguredOption("false")
```

Comment:
Is this the right default? I think it would be more convenient for users if redirects were followed by default.

Reply:
I was using the same default as there is with reactive WebClient.